src/main/java/it/gov/pagopa/pu/classification/model/PaymentsNotification.java

```diff
@@ -0,0 +1,72 @@
+package it.gov.pagopa.pu.classification.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import java.util.Date;
+import lombok.*;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "payments_notification")
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
+@EqualsAndHashCode(of = "paymentsNotificationId", callSuper = false)
+public class PaymentsNotification extends BaseEntity implements Serializable{
+
+  @Id
+  private String paymentsNotificationId;
```

Comment:
please, align with table name
```suggestion
@EqualsAndHashCode(of = "paymentNotificationId", callSuper = false)
public class PaymentNotification extends BaseEntity implements Serializable{

  @Id
  private String paymentNotificationId;
```